db/fixtures/miq_product_features.yml

```diff
@@ -3743,6 +3743,10 @@
       :description: Show Ad hoc Metrics of Providers
       :feature_type: view
       :identifier: ems_container_ad_hoc_metrics
+    - :name: Capture Metrics
+      :description: Capture metrics related to this Containers Provider
+      :feature_type: view
+      :identifier: ems_container_capture_metrics
```

Comment:
That's interesting.  I don't see a similar product feature for other providers.  The screenshot doesn't lie about it though 🤣   I wonder if the other provider types have a capture metrics action that will also raise an error.

Reply:
i see this route in ui-classic repo for one of the actions in toolbar.

Reply:
Ok, I was just curious if there are other broken pages for the other providers.